docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`770` Adds loading screen while waiting for the backend to start.
```

Comment:
Can you add major here? It's something stupid forced on us due to the releases package that I have to fix.

Reply:
Does it also have minor?

Reply:
Don't remember. It's just some rules that need to be followed. I have an issue to handle this one way or the other at some point since it's unsustainable as a way of writing the changelog: https://github.com/rotki/rotki/issues/704